src/main/java/meditracker/dailymedication/DailyMedicationManager.java

```diff
@@ -50,6 +51,7 @@ public static void createDailyMedicationManager() {
      * @param lines lines of String read from each row in the textfile
      */
     public static void importDailyMedicationManager(List<String> lines) {
+        clearDailyMedication(); // For when loading from save file
```

Comment:
Oh this was my fault, I forgot I removed them when I merged my branch. Thanks for catching my mistake!